src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/directionsquestionnaire/HearingContentProvider.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content.directionsquestionnaire;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.ExpertReportContent;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.HearingContent;
+import uk.gov.hmcts.cmc.claimstore.utils.DateUtils;
+import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.DirectionsQuestionnaire;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.ExpertReport;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.ExpertRequest;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.RequireSupport;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.UnavailableDate;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+import static java.util.stream.Collectors.toList;
+import static uk.gov.hmcts.cmc.ccd.util.StreamUtil.asStream;
+
+@Component
+public class HearingContentProvider {
+
+    private static final String DISABLED_ACCESS = "Disabled Access";
+    private static final String YES = "YES";
+    private static final String NO = "NO";
```

Comment:
Are you sure we want these as capitals in PDF? We are using `Yes` / `No` in other PDFs

Reply:
Will Sort it. I think when I moved to constant, I copy pased I think. :-) My mistake.